iOS/MainFeed/MainFeedViewController.swift

```diff
@@ -699,6 +699,38 @@ class MainFeedViewController: UITableViewController, UndoableCommandRunner {
 		}
 	}
 	
+	func updateNavigationBarSubtitle() {
+		let progress = AccountManager.shared.combinedRefreshProgress
+
+		if progress.isComplete {
+			if let accountLastArticleFetchEndTime = AccountManager.shared.lastArticleFetchEndTime {
+				if Date() > accountLastArticleFetchEndTime.addingTimeInterval(60) {
+					let relativeDateTimeFormatter = RelativeDateTimeFormatter()
+					relativeDateTimeFormatter.dateTimeStyle = .named
+					let refreshed = relativeDateTimeFormatter.localizedString(for: accountLastArticleFetchEndTime, relativeTo: Date())
+					let localizedRefreshText = NSLocalizedString("Updated %@", comment: "Updated")
+					let refreshText = NSString.localizedStringWithFormat(localizedRefreshText as NSString, refreshed) as String
+					navigationController?.navigationBar.topItem?.subtitle = refreshText
+				} else {
+					navigationController?.navigationBar.topItem?.subtitle = NSLocalizedString("Updated Just Now", comment: "Updated Just Now")
+				}
+			} else {
+				navigationController?.navigationBar.topItem?.subtitle = ""
+			}
+
+		} else {
+			navigationController?.navigationBar.topItem?.subtitle = NSLocalizedString("Updating...", comment: "Updating...")
+		}
+		
+		scheduleNavigationBarSubtitleUpdate()
```

Comment:
Would you add a check so that it doesn’t schedule an update when an update is already scheduled? Or maybe just make this a timer scheduled for once a minute? (As written, if there are several calls to updateNavigationBarSubtitle in a row for whatever reason, then it would schedule multiple updates.)

This can be done in a follow-up PR, of course.